card/lib/card/cache.rb

```diff
@@ -40,6 +39,11 @@ def renew
         reset_local
       end
 
+      def prefix_root
+        @@prefix_root ||= ( cfg = Cardio.config and cfg = cfg.database_configuration and
+                            cfg[Rails.env]['database'] )
+      end
+
```

Comment:
I guess I was mostly looking for Wagn at first.  Need to survey those a bit more.